scrc/preprocessors/extractors/judgment_extractor.py

```diff
@@ -65,7 +65,7 @@ def select_df(self, engine: str, spider: str) -> str:
         return self.select(engine,
                            f"section {join_decision_and_language_on_parameter('decision_id', 'section.decision_id')} {join_file_on_decision()}",
                            f"section.decision_id, section_text, '{spider}' as spider, iso_code as language, html_url",
-                           where=f"section.section_type_id = 5 AND section.decision_id IN {where_string_spider('decision_id', spider)} {only_given_decision_ids_string}",
+                           where=f"section.section_type_id = 6 AND section.decision_id IN {where_string_spider('decision_id', spider)} {only_given_decision_ids_string}",
```

Comment:
here it probably makes sense to replace this magic number with the correct variable from the Section enum, right?

Reply:
I replaced it to always use the Section.RULING.value